src/components/SendTokenDialog.vue

```diff
@@ -648,6 +681,81 @@ export default defineComponent({
       this.showDeleteDialog = false;
       this.clearAllWorkers();
     },
+    writeTokensToCard: function () {
+      if (!this.scanningCard) {
+        try {
+          this.ndef = new NDEFReader();
+          this.controller = new AbortController();
+          const signal = this.controller.signal;
+          this.ndef
+            .scan({ signal })
+            .then(() => {
+              console.log("> Scan started");
+
+              this.ndef.onreadingerror = (error) => {
+                console.error(`Cannot read NDEF data! ${error}`);
+                notifyError("Cannot read data from the NFC tag");
+                this.controller.abort();
+                this.scanningCard = false;
+              };
+
+              this.ndef.onreading = ({ message, serialNumber }) => {
+                console.log(`Read card ${serialNumber}`);
+                this.controller.abort();
+                this.scanningCard = false;
+                try {
+                  const tokenURL =
+                    window.location.toString() +
+                    "#token=" +
+                    this.sendData.tokensBase64;
+                  this.ndef
+                    .write(
+                      {
+                        records: [
+                          {
+                            recordType: "url",
+                            data: tokenURL,
+                          },
+                        ],
+                      },
+                      {
+                        overwrite: true,
+                      }
+                    )
+                    .then(() => {
+                      console.log("Successfully flashed tokens to card!");
+                      notifySuccess("Successfully flashed tokens to card!");
+                      this.showSendTokens = false;
+                    })
+                    .catch((err) => {
+                      console.error(`Argh! ${err}`);
+                      notifyError(`Argh! ${err}`);
+                    });
+                } catch (err) {
+                  console.error(`Argh! ${err}`);
+                  notifyError(`Argh! ${err}`);
+                }
+              };
+              this.scanningCard = true;
+            })
+            .catch((error) => {
+              console.error(`Argh! ${error}`);
+              notifyError(`Argh! ${error}`);
+              this.scanningCard = false;
+            });
+          notifyWarning("This will OVERWRITE your card!");
```

Comment:
nitpick, maybe being more explicit helps?

```suggestion
          notifyWarning("Tap an NFC card ... This action will OVERWRITE the contents!");
```